component/site/libraries/iCalEvent.php

```diff
@@ -64,6 +64,13 @@ function iCalEvent( &$db ) {
 	function store($updateNulls=false , $overwriteCreator = false) {
 		$user = JFactory::getUser();
 
+		$jinput = JFactory::getApplication()->input;
+		$curr_task = $jinput->get('task');
```

Comment:
By default, it is cmd. So cleaned as per CMD. I will update it anyhow to make it clear.